1477 - Edits a focus issue

jlandiseigsti opened the pull request:
The accessibility ticket put the focus on the newly created field. However, it was firing in places we didn't want; this limits the scope so it only fires when we want it.

mtribone commented:
<img width="1124" alt="Screen Shot 2019-07-26 at 10 34 55 AM" src="https://user-images.githubusercontent.com/4163828/61959291-15a89800-af91-11e9-90d8-cad730c195e8.png">

awead commented on app/views/curation_concerns/base/_work_readme.html.erb:
```diff
@@ -2,7 +2,7 @@
-    <p><%= presenter.readme_prompt %></p>
+  <% elsif presenter.show_readme_prompt? %>
+    <p><%= I18n.t('scholarsphere.generic_work.readme_text_html').html_safe %></p>
   <% end %>
```
@mtribone we can change this whole if/elsif thing to just:
```
<% if presenter.readme.present? %>
  <%= raw(presenter.readme) %>
<% else %>
  <p><%= I18n.t('scholarsphere.generic_work.readme_text_html').html_safe %></p>
<% end %>

jlandiseigsti committed:
1477 - Edits a focus issue

The accessibility ticket put the focus on the newly created field. However,
it was firing in places we didn't want; this limits the scope so it only fires when we want it.
